src/touchstone/databases/prom_config.yaml

```
@@ -0,0 +1,29 @@
+---
+- url: http://localhost:9090
+  query_list:
+    - node_disk_io_time_seconds_total
+    - up
+  bearer_token:
+  disable_ssl: True
+  start_time_list:
+  end_time_list:
+
+
+- url: https://prometheus-k8s-openshift-monitoring.apps.awsovn4312aakarsh.perf-testing.devcluster.openshift.com
+  query_list:
+    - sum(container_memory_rss{namespace=~"openshift-kube-apiserver",name!="",container=~"kube-apiserver.*"}) by (container)
+    - up
+  bearer_token: eyJhbGciOiJSUzI1NiIsImtpZCI6Ii1tS1piNmhXY2p6UXd3WFVwejJCZlREQ01HeGdzRmhyaTAxcDVhNUlmZ0UifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uY
```

This should be multi line to avoid confusion:

```suggestion
  bearer_token: > eyJhbGciOiJSUzI1NiIsImtpZCI6Ii1tS1piNmhXY2p6UXd3WFVwejJCZlREQ01HeGdzRmhyaTAxcDVhNUlmZ0UifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uY
```